.github/workflows/pr.yaml

```diff
@@ -224,6 +224,11 @@ jobs:
       - name: Run Unit Tests
         run: |
           make test-unit
+      - name: Upload coverage reports to Codecov
+        uses: codecov/codecov-action@v4.0.1
```

Comment:
Would you mind using this pattern?

`...@hash # vX.X.X`

https://github.com/aquasecurity/tracee/blob/00598de5cd314b6871e06d6c561f6a7708923f04/.github/workflows/pr.yaml#L128